content/500-platform/50-support.mdx

```diff
@@ -7,26 +7,32 @@ toc: true
 displayed_sidebar: platformSidebar
 ---
 
-<TopBlock>
-
 Your feedback is invaluable, and we encourage you to share your experiences with us on [Discord](https://pris.ly/discord).
 
-</TopBlock>
 
-### Community Support
+## Support
+
+### Community support
 
 Reach out to us in our [Discord](https://pris.ly/discord).
 
-### Standard Support
+### Standard support
 
-- Email support, support@prisma.io
+- Email support, <a href="mailto:support@prisma.io">support@prisma.io</a>
 - Mon-Fri, 9am-5pm CET
 
-### Premium Support
+### Premium support
 
-- Email support, support@prisma.io
+- Email support, <a href="mailto:support@prisma.io">support@prisma.io</a>
 - 24/7
 
-### Dedicated Support
+### Dedicated support
 
 Dedicated contact person.
+
+## Deleting your PDP account
+
+
+If you want to delete your PDP account, **email us at <a href="mailto:support@prisma.io">support@prisma.io</a>**. 
```

Comment:
I'd remove this, so we don't have to update it when we implement this. We can link to the Console, where these instructions also show up.

Reply:
> I'd remove this,

What exactly does "this" refer to here? is it the entire section **Deleting your PDP account** or just this sentence "If you want to delete your PDP account, **email us at <a href="mailto:support@prisma.io">support@prisma.io</a>**."?

FWIW, this was a request from Nurul because he needs to explain this to people over and over.

Reply:
I also should have been more clear: 

The thing that Nurul told me he needs to explain frequently is that people need to  explicitly disable Accelerate and Pulse in _all_ environments of _all_ projects, so that's what we should be documenting.

However, that hint feels off to me if we don't also mention here how an account is being deleted. 

My suggestion would be:
- keep everything as is
- delete the entire `## Deleting your PDP account` section

Any preferences @petradonka @nurul3101?

Reply:
It's fine, lets leave it as-is then. I can see how that'd be useful. I do doubt that people will come here to read up on this, but it doesn't hurt to have it.